src/index.html

```diff
@@ -1,15 +1,14 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>jeteo [dev]</title>
-</head>
-
-<body>
-    <main id="root"></main>
-</body>
+<!DOCTYPE HTML>
+<html lang="pl">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./index.css" type="css" />
+        <title>Jeteo [dev]</title>
+    </head>
 
+    <body>
+        <div style="width: 100%; height: 100%;" id="root"></div>
```

Comment:
`<main>` tag wasn't here for no reason
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

our webpack config doesn't support (and probably never will) any global id/class names (`css as modules` is default)

Reply:
still... this styles shouldn't be here